lib/ronin/recon/builtin/web/screenshot.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+#
+# ronin-recon - A micro-framework and tool for performing reconnaissance.
+#
+# Copyright (c) 2023-2024 Hal Brodigan (postmodern.mod3@gmail.com)
+#
+# ronin-recon is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ronin-recon is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with ronin-recon.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+require 'ronin/recon/worker'
+
+require 'ronin/web/browser'
+
+module Ronin
+  module Recon
+    module Web
+      #
+      # A recon worker that takes a screenshot of a page.
+      #
+      class Screenshot < Worker
+        register 'web/screenshot'
+
+        summary 'Visits a website and takes a screenshot of it'
+        description <<~DESC
+          Visits a website and takes a screenshot of it
+        DESC
+
+        accepts URL
+        outputs URL # TODO: REMOVE
+
+        param :output_path, String, required: true,
+                                    desc:     'URL of the website you want to screenshot'
+
+        #
+        # Visits a website and takes a screenshot of it.
+        #
+        # @param [Values::URL] url
+        #   The URL of the website you want to screenshot.
+        #
+        def process(url)
+          Ronin::Web::Browser.open do |browser|
```

Comment:
We probably want to initialize the `Ronin::Web::Browser` in `initialize`, so we don't have to open a new browser for each URL.